blog/2024-11-26-robotics-mongodb/index.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "MongoDB vs ReductStore: Choosing the Right Database for Robotics Applications"
+description: TODO
+authors: gracija
+tags: [comparison, mongodb, robotics]
+slug: robotics-mongodb-vs-reductstore
+date: 2024-11-26
+image: ./img/introduction_diagram.drawio.png
+---
+
+![TODO](./img/introduction_diagram.drawio.png)
+
+Robotics applications generate and process a wide variety of data, such as sensor readings, video streams, logs, and AI model outputs. Managing this data efficiently is crucial because it affects the performance, scalability, and reliability of the entire system.
+
+In this article, we'll compare [**ReductStore**](/) and **MongoDB**, two databases designed to handle different aspects of data management. ReductStore is a time-series blob storage solution optimized for managing large amounts of data coming from continuous streams. MongoDB, on the other hand, is a popular NoSQL database known for its flexibility, scalability, and support for unstructured and semi-structured data.
+
+By understanding the strengths and limitations of each, you can make an informed decision to meet your project's specific data needs.
+
+{/* truncate */}
+
+**TL;DR;**
+
+| **Feature**                    | **ReductStore**                                                                           | **MongoDB**                                                                        |
+| ------------------------------ | ----------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| **Architecture**               | Buckets, entries, records, and blocks                                                     | Collections of JSON-like documents                                                 |
+| **Query Language**             | Query data via CLI, SDKs, or HTTP API                                                     | MongoDB Query Language (MQL)                                                       |
```

Comment:
in the current version, we don't have any query language. I would say something like:

Data Queering:
* ReductStore: query requests via time ranges with filters
* MongoDB Query Language (MQL)